package.json

```diff
@@ -38,6 +38,7 @@
     "gulp-util": "^3.0.8",
     "mkdirp": "^0.5.1",
     "moment": "^2.22.1",
+    "natives": "^1.1.6",
```

Comment:
I can't see where this is used - but we should definitely chat about it!

Reply:
I introduced this after the pages failed to render and it was suggested has a fix on GitHub, which did work However I have removed, deleted the node modules and rebuilt and it's working without.